### ISSUE TITLE
Any plans to release pre-trained Small ImageNet model? 

### ISSUE THREAD

**Issue by alexggmatthews** [openai/pixel-cnn#8]
This Repo is a great resource. Thank you very much. 

I notice the readme mentions results on Small ImageNet . Are there any plans to release a pre-trained model as you've done with CIFAR 10, please? What does the ImageNet model achieve in Bits Per Dim on the validation set?
